client/src/chat/components/ChatScreen.tsx

```diff
@@ -14,16 +14,23 @@ import {
 	useUserChatData,
 } from "../ChatContext";
 import { chatColors } from "@/chatColors";
-import { BsCircleFill, BsThreeDotsVertical, BsSendFill } from "react-icons/bs";
+import { useThreadImage, useThreadLabel } from "@/utils";
+import { BsCircleFill, BsThreeDotsVertical, BsSendFill, BsFillEmojiSunglassesFill } from "react-icons/bs";
 import { GrAttachment } from "react-icons/gr";
 import { useWebSocket } from "react-use-websocket/dist/lib/use-websocket";
 import { DateTime, Duration } from "luxon";
-import { useThreadImage, useThreadLabel } from "@/utils";
+import data from '@emoji-mart/data';
+import Picker from '@emoji-mart/react';
+import "./EmojiPickerStyles.css";
+import useOutsideClick from "./hooks/useOutsideClick";
 
 interface ChatMessageProps {
 	data: Message;
 	prev: Message | undefined;
 }
+interface SyntheticEvent {
```

Comment:
Better to avoid these sorts of re-writes of intermediate types used by libraries - either use their definition, or use the value inline (more below)